sodium/src/secbuf.rs

```diff
@@ -53,7 +58,7 @@ struct SodiumBuf {
 impl Bufferable for SodiumBuf {
     /// warning: funky sizes may result in mis-alignment
     fn new(s: usize) -> Box<Bufferable> {
-        if s != 16 && s != 32 && s != 64 {
+        if s != 8 && s != 16 && s != 32 && s != 64 {
```

Comment:
mmm, unless we want to get fancy and actually check the alignment, I *think* 16 is the minimum on a 64 bit machine... 8 might be mis-aligned. (Somebody correct me if I'm wrong, alignment is a little out of my depth - https://en.wikipedia.org/wiki/Data_structure_alignment)